source/docs/hardware/sensors/gyros-hardware.rst

```diff
@@ -14,22 +14,24 @@ Several popular FRC devices (known as "inertial measurement units," or "IMUs") c
 Types of gyros
 --------------
 
-There are two types of Gyros commonly-used in FRC: single-axis gyros, and three-axis gyros.
+There are two types of Gyros commonly-used in FRC: single-axis gyros, three-axis gyros and IMUs, which often include a 3-axis gyro.
 
 Single-axis gyros
 ^^^^^^^^^^^^^^^^^
 
-.. todo:: add picture
+.. image:: images/gyros-hardware/analog-devices-frc-gyro-to-roborio.svg
 
-As per their name, single-axis gyros measure rotation rate around a single axis.  This axis is generally specified on the physical device, and mounting the device in the proper orientation so that the desired axis is measured is highly important.  Single-axis gyros generally output an analog voltage corresponding to the measured rate of rotation, and so connect to the roboRIO's :doc:`analog input <analog-inputs-hardware>` ports.
+As per their name, single-axis gyros measure rotation rate around a single axis.  This axis is generally specified on the physical device, and mounting the device in the proper orientation so that the desired axis is measured is highly important.  Some single-axis gyros can output an analog voltage corresponding to the measured rate of rotation, and so connect to the roboRIO's :doc:`analog input <analog-inputs-hardware>` ports. Other single-axis gyros, such as as the `ADXRS450 <https://wiki.analog.com/first/first_robotics_donation_resources>`__ pictured above, use the :ref:`SPI port <docs/hardware/sensors/serial-buses:SPI>` on the roboRIO instead.
 
 The `Analog Devices ADXRS450 FRC Gyro Board <https://www.analog.com/en/landing-pages/001/first.html>`__ that has been in FIRST Choice in recent years is a commonly used single axis gyro.
 
 Three-axis gyros
 ^^^^^^^^^^^^^^^^
 
-.. todo:: add picture
+.. image:: images/gyros-hardware/triple-axis-i2c-gyro-to-roborio.svg
 
-Three-axis gyros measure rotation rate around all three spacial axes (typically labeled x, y, and z).
+.. image:: images/gyros-hardware/yaw-pitch-roll.png
```

Comment:
> .. image:: images/gyros-hardware/yaw-pitch-roll.png

Move this below the pitch, yaw, and roll line for better flow.

Reply:
Fixed in aa03d65.